notebooks/407-pedestrian-tracking-webcam/README.md

```diff
@@ -0,0 +1,24 @@
+# Live Pedestrian Tracking with OpenVINO™
```

Comment:
maybe better person tracking?
pedestrian has some assumption that it will be used only in road scenario with video from car video registrator, but generally if you assume that it can be run using webcam, of course, nobody go to the crossroad with notebook, it is just unsafe :)

Reply:
Sure I will update it later